internal/clickhouseclient/jsoncompactstrings.go

```diff
@@ -59,7 +60,7 @@ func (j jsonCompatStrings) Rows() []Row {
 					data.Set(colNames[i], val)
 				}
 			default:
-				data.Set(colNames[i], field)
+				panic(fmt.Sprintf("unknown data type %q", colTypes[i]))
```

Comment:
we can't return an error from this function and we definitely don't want this to fail silently.
The success of this code path does not depend on user input, so the only way we can panic is when there is a bug in our code.